core/internal/protohelpers.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/any"
 
-	tpb "github.com/google/keytransparency/core/proto/keytransparency_v1_types"
+	tpb "github.com/google/keytransparency/core/proto/keytransparency_v1"
```

Comment:
nit: use pb package alias, as you've done elsewhere, and since there 'no _type' suffix any more.

Reply:
done